project-accessibility on request Mathilda

Code-lab-web opened the pull request.

Code-lab-web committed:
upload code

Code-lab-web committed:
upload of code

Code-lab-web committed:
git commit -m "upload of code"

Code-lab-web committed:
upload of code

Code-lab-web committed:
git add .

Code-lab-web committed:
git commit -m "Code-lab-web"

Code-lab-web committed:
Add redirects for various URLs in _rediects.md

Commit:
git commit -m "Code-lab-web"

Code-lab-web committed:
Update package.json to add MUI and styled-components dependencies

Code-lab-web committed:
Add @fontsource/roboto dependency to package.json and package-lock.json

Code-lab-web committed:
Add Roboto font and Material Icons dependencies to styles and package files

Code-lab-web committed:
Add React and ReactDOM dependencies to package.json and package-lock.json

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit-m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code-lab-web"

Commit:
Code-lab-web

Commit:
git commit -m "Code-lab-web"

Commit:
git commit -m "Code.lab"

JennieDalgren reviewed (changes requested):
Read the requirements for this project and follow the guide on how to work with github! It is not possible to review this at all.

You are not approved on this project

servis committed:
git commit -m "Code-lab-web"

servis committed:
git commit -m "Code-lab-web"

HIPPIEKICK commented:
ping

Code-lab-web committed:
Create SECURITY.md for security policy

Added a security policy document outlining supported versions and vulnerability reporting.

Code-lab-web committed:
Merge branch 'Technigo:main' into main
